cert-manager/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Build the manager binary
+FROM golang:1.19 as builder
```

Comment:
Any reason why you don't use alpine here?

Reply:
just left the aurogenerated file , I will change it :)